feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.html

```diff
@@ -17,31 +17,34 @@
   let-level="level"
   let-idPrefix="idPrefix"
 >
-  <ng-container *ngFor="let group of overviewGroups">
-    <div [ngClass]="getGroupLevelStyleClasses(level)">
-      <button
-        class="cx-menu-item"
-        role="tab"
-        [attr.aria-label]="
-          'configurator.a11y.groupName'
-            | cxTranslate: { group: group.groupDescription }
-        "
-        (click)="navigateToGroup(idPrefix, group.id)"
-      >
-        <span aria-hidden="true"> {{ group.groupDescription }}</span>
-      </button>
-      <ng-container *ngIf="group.subGroups?.length > 0">
-        <ng-container
-          *ngTemplateOutlet="
-            groups;
-            context: {
-              overviewGroups: group.subGroups,
-              level: level + 1,
-              idPrefix: idPrefix + group.id
-            }
+  <ul>
+    <ng-container *ngFor="let group of overviewGroups">
+      <li [ngClass]="getGroupLevelStyleClasses(level)">
+        <button
+          id="{{ getMenuItemId(idPrefix, group.id) }}"
+          class="cx-menu-item"
+          [attr.aria-label]="
+            'configurator.a11y.groupName'
+              | cxTranslate: { group: group.groupDescription }
           "
-        ></ng-container>
-      </ng-container>
-    </div>
-  </ng-container>
+          (click)="navigateToGroup(idPrefix, group.id)"
+        >
+          <span aria-hidden="true"> {{ group.groupDescription }}</span>
+          <cx-icon [type]="iconTypes.ARROW_LEFT" aria-hidden="true"></cx-icon>
+        </button>
+        <ng-container *ngIf="group.subGroups?.length > 0">
+          <ng-container
+            *ngTemplateOutlet="
+              groups;
+              context: {
+                overviewGroups: group.subGroups,
+                level: level + 1,
+                idPrefix: idPrefix ? idPrefix + '--' + group.id : group.id
```

Comment:
line is duplicated in configurator-oveview maybe we can move logic to configurator-storefront-utils

Reply:
It's true but it is how the context of the template is normally defined.
The question is whether it will be more readable if the context will be extracted to the helper class.
I don't think so then one has to jump to the helper class to better understand what context will be forwarded at this play.
I would suggest to let it be as it is.

Reply:
I extracted to a separate method `getPrefixId(idPrefix, group.id)`

Reply:
What about it?